runtime/makelib/targets.mk.ztpf.inc.ftl

```diff
@@ -118,9 +118,9 @@ CFLAGS += $(J9M31) -DS390 -D_LONG_LONG -DJ9VM_TIERED_CODE_CACHE -fno-strict-alia
 CXXFLAGS += $(J9M31) -DS390 -D_LONG_LONG -DJ9VM_TIERED_CODE_CACHE -fno-strict-aliasing
 CPPFLAGS += -DS390 -D_LONG_LONG -DJ9VM_TIERED_CODE_CACHE
 ifdef j9vm_env_data64
-  CFLAGS += -DS39064
-  CXXFLAGS += -DS39064
-  CPPFLAGS += -DS39064
+  CFLAGS += -DS39064 -fstack-protector
```

Comment:
@jjohnst-us fyi

Reply:
Appreciate the heads-up Peter! 
@fengxue-IS  we do not have ssp support implemented in the z/tpf version of glibc. We would have to implement both __stack_chk_fail and __stack_chk_guard. That is not something we have identified as a future work item - we do have a system option so that our kernel will check/walk the stack before dispatching a thread. This is done before the function call and before the return to caller.

Reply:
@pelemie thanks for this information.
@pshipton should I remove the change for z/TPF?

Reply:
@fengxue-IS yes

Reply:
Yes please! And, thank you.

Reply:
fixed